projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -103,8 +103,21 @@ export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
     @Input()
     canOpen = true;
 
+    private readonly onHoverTarget$ = new Subject<HTMLElement>();
+
+    private readonly _hover$ = combineLatest([
```

Comment:
I don't like underscore names, let's name it `hostHover$` instead so we know it's about host being hovered.